app/controllers/hostgroups_controller.rb

```diff
@@ -102,8 +103,31 @@ def process_hostgroup
     render :partial => "form"
   end
 
+  def taxonomy_warnings
+    @warnings ||= []
+
+    @warnings += organization_warnings(@hostgroup)
+    @warnings += location_warnings(@hostgroup)
+  end
+
   private
 
+  def organization_warnings(hostgroup)
+    warnings = []
+
+    organizations_count = @hostgroup.hosts.unscoped.distinct.count(:organization_id)
+    warnings << _('The hostgroup is used by hosts in multiple organizations') if organizations_count > 1
```

Comment:
Can we provide a clickable link for 'hosts' that lists all hosts by `@hostgroup.hosts` and another for `multiple organizations` that lists `@hostgroup.hosts.map(&:organization).map(&:label).uniq`? This would allow users to understand the problem better